src/include/miopen/db_record.hpp

```diff
@@ -70,14 +70,20 @@ class DbRecord
 {
 public:
     template <class TValue>
-    class Iterator : public std::iterator<std::input_iterator_tag, std::pair<std::string, TValue>>
```

Comment:
@DrizztDoUrden Please review this change and the one below. Thanks!

Reply:
It's okay. I don't think it matters if it is derived from anything or not.